src/Form.php

```diff
@@ -244,10 +247,12 @@ public function setModel(Model $model, $fields = null)
 
     /**
      * Adds callback in submit hook.
+     *
+     * @return $this
      */
-    public function onSubmit(callable $callback)
+    public function onSubmit(\Closure $callback)
```

Comment:
This is probably the only method where alias for `$this->onHook(self::HOOK_SUBMIT, $callback);` makes sense.